client/src/components/Navigation/styles.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
```

Comment:
Sorry to nitpick, but let's call this file:

"Login.styles.ts"

This way, we specify that it is the style-sheet for the Login component. Also, we want to name is TS and not TSX because we aren't using JSX in the file. 

Make sure to also make this change with the other style-sheet file.

Reply:
no need to apologize at all. I need all the nitpickiness you can give haha I'll gladly take critiques to better my code for the future.

I will make these changes and get it pushed